static/cms/admin/config.yml

```diff
@@ -235,5 +235,63 @@ collections:
           - { label: 'Maltese', value: 'mt' }
           - { label: 'Welsh', value: 'cy' }
           - { label: 'Belarusian', value: 'be' }
+  - name: events
+    label: Events
+    label_singular: Event
+    folder: src/collections/events
+    slug: '{{title}}'
+    create: true
+    fields:
+      - name: title
+        label: Title
+      - name: url
+        label: URL
+      - name: category
+        label: Category
+        widget: select
+        options:
+          - { label: 'Tournament', value: 'tournament' }
+          - { label: 'Social', value: 'social' }
+          - { label: 'Learning', value: 'learning' }
+          - { label: 'Generic', value: 'generic' }
+      - name: startDateTime
+        label: Start Date/Time
+        widget: datetime
+        format: YYYY-MM-DDTHH:mm:ss
+        picker_utc: true
+        default:
+      - name: ignoreStartTime
+        label: Ignore Start Time
+        widget: boolean
+        required: false
```

Comment:
Perhaps `default: false` could be good to force a value on these. It may be good to invert the logic, as well.

With that I mean instead of `ignoreStartTime` it might be `useStartTime` and all checks inverted. That seems easier and less confusing on the form.

Reply:
Fair enough - will change